microsoft-365/compliance/sit-edm-wizard.md

```diff
@@ -49,40 +49,55 @@ steps in [Part 1: Set up EDM-based classification](create-custom-sensitive-infor
 
-14. Choose **Next** and fill in a **Name** and **Description for admins**.
+7.  If none of the built-in sensitive info types will match the data in the column you selected you will have to create a custom SIT that does. If you selected the Ignored Delimiters option for this column when defining your schema, make sure the custom SIT you create will match data with and without the selected delimiters. Also, avoid creating a custom SIT that would match too much unrelated text (e.g. one that uses a regular expression that matches any random words or string) since this can cause the system to temporariluy stop matching data. See [Sensitive Information Type Entity Definitions](sensitive-information-type-entity-definitions.md) to learn more about the available sensitive information types and [Get started with custom sensitive information types](create-a-custom-sensitive-information-type.md) to learn about creating a custom sensitive infromation type. 
 
-15. Review and choose **Submit**.
+8.  Choose one or more columns in your schema to be used as "Supporting elements" and whether all or some of the elements must be present to generate a match. When you select supporting elements, values for the selected columns in the same row as the primary element that was found must be found close to each other in the content. You can also select a minimum and maximum number of additional elements to require.
+
+9. Choose **Done**.
+
+10. Choose **Create pattern** if you want to creaet additional patterns for your EDM sensitive info type, otherwise choose **Next**.
+
+11. Choose your desired **Confidence level and character proximity**. This will be the default value for the whole EDM sensitive info type.
+
+12. Choose **Next** and fill in a **Name** and **Description for admins**. Choose **Next**.
+
+13. Review and choose **Submit** to confirm the creation of your EDM Sensitive Info Type.
 
 You can delete or edit the sensitive information type pattern by selecting it which surfaces the edit and delete controls.
 
+After creating an EDM sensitive information type and verifying that your sensitive information table has correctly been uploaded, you can test that it's working properly by selecting the EDM sensitive info type and selecting the Test option on the pane that appears on the right. You will be asked to submit a document that contains data you want to try to detect, and after the file has been uploaded and scanned you will see if there were matches to any of the patterns defined in your EDM type. If you don't find any matches, try the following:
+- Confirm that your sensitive data was uploaded correctly using the commands explained in [the guidance for uploading your sensitive data using the EDM tool](links-to-edm-upload-agent-by-subscription-type.md).
```

Comment:
Please confirm that this link is correct. When selecting it, I get a 404 error.

Reply:
@JohanFreelancer9 
Apologies, not too skilled with editing in Github. The correct link is: 
create-custom-sensitive-information-types-with-exact-data-match-based-classification.md#links-to-edm-upload-agent-by-subscription-type
But since I don't know how to make this edit without interfering with your edits (do I need to create a separate pull request? I just don't want to add more work for you unnecessarily) maybe you can just change the URL directly?

Reply:
@Esaggese - thank you for committing my suggested changes. Please feel free to edit the link. It is no problem at all.